### PR TITLE
Add extra logging when remounting NFS volume on Windows.

### PR DESCRIPTION
**Pull request by Dor1s**

### CODE REVIEW

File: src/python/bot/init_scripts/windows.py [hunk 1]
```diff
@@ -65,6 +66,7 @@ def remount_if_needed():
   subprocess.call(['umount', '-f', nfs_root])
 
   # Mount the nfs drive.
+  logs.log_warn('Trying to remount the NFS volume.')
```
Comment (mbarbella-chromium) [inline src/python/bot/init_scripts/windows.py]: I think it's probably a bit safer to move this to 85. I'm not sure if the startup script mounts and creates the file, but if not we expect this to happen once.

Comment (Dor1s) [reply src/python/bot/init_scripts/windows.py]: I'm afraid that `subprocess.check_call` on line 72 may raise an exception, plus I think the startup script does mount the volume too.

Comment (mbarbella-chromium) [reply src/python/bot/init_scripts/windows.py]: Seems fine to keep it as-is then.

**Pull request merged**